packages/asset/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sandbox-smart-contracts/asset",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
How will we make sure that these versions are in line with the index being used to reinitialize?

Reply:
We can't and we won't, the NPM version will mismatch anyway.
Imagine we upgrade Asset to version 2, and then in a subsequent upgrade we move asset to v3 and catalyst to v2.

Because we have multiple contracts in the package that can be upgraded independently we should not be trying to make the npm version match the contract version.